package.json

```diff
@@ -392,6 +474,99 @@
             "default": ""
           }
         }
+      },
+      {
+        "type": "object",
+        "title": "vscode-dfdl",
+        "properties": {
+          "vscode-dfdl.enabled": {
+            "type": "boolean",
+            "default": true,
+            "description": "Enables the highlighting and status bar",
+            "scope": "resource"
+          },
+          "vscode-dfdl.showPath": {
+            "type": "boolean",
+            "default": true,
+            "description": "Enables showing tag's path in status bar",
+            "scope": "resource"
+          },
+          "vscode-dfdl.showRuler": {
+            "type": "boolean",
+            "default": true,
+            "description": "Enables showing highlighted tag pair in ruler section",
+            "scope": "resource"
+          },
+          "vscode-dfdl.highlightSelfClosing": {
+            "type": "boolean",
+            "default": false,
+            "description": "Should self-closing tags be highlighted",
+            "scope": "resource"
+          },
+          "vscode-dfdl.highlightFromContent": {
+            "type": "boolean",
+            "default": true,
+            "description": "Whether to highlight matching tag from inside the tag content",
+            "scope": "resource"
+          },
+          "vscode-dfdl.highlightFromName": {
+            "type": "boolean",
+            "default": true,
+            "description": "Whether to highlight matching tag from the tag name",
+            "scope": "resource"
+          },
+          "vscode-dfdl.highlightFromAttributes": {
+            "type": "boolean",
+            "default": true,
+            "description": "Whether to highlight matching tag from the tag attributes",
+            "scope": "resource"
+          },
+          "vscode-dfdl.noDefaultEmptyElements": {
+            "type": "boolean",
+            "default": false,
+            "description": "Don't use default HTML empty elements",
+            "scope": "resource"
+          },
+          "vscode-dfdl.customEmptyElements": {
+            "type": "array",
+            "default": null,
+            "description": "Custom empty elements in addition to the default HTML empty elements",
+            "scope": "resource"
+          },
+          "vscode-dfdl.styles": {
```

Comment:
@stevedlawrence For me it didn't it just used my default color theme

Reply:
@rthomas320 would be best to explain I imagine because I have tried the extension with multiple themes and no colors seemed to be broken so I am not sure if it affects anything currently or only in a specific way that its hard to notice

Reply:
@michael-hoke implemented most of the colorization.  I believe he made a comment about color theming for the xpath statements, but I can't recall exactly what he said.  Maybe he can provide additional detail.